pkg/specgen/volumes.go

```diff
@@ -180,10 +180,20 @@ func GenVolumeMounts(volumeFlag []string) (map[string]spec.Mount, map[string]*Na
 			}
 		} else {
 			// This is a named volume
+			// however, we might still have a subpath
+			// the syntax is NAMED_VOL/sub/path:dest
+
+			subpath := ""
+			srcAndSubPath := strings.SplitN(src, "/", 2)
```

Comment:
Is this to add CLI support for subpaths? I would prefer it to be an option, as opposed to `volumename/subpath:/path/in/container` (especially given that the first syntax could be a relative path, not a named volume)

Reply:
Yes subpath makes sense as an option.  As I am looking at the quadlet PR.

Reply:
@mheon so you'd prefer a --subpath flag? and it would be a string array to correspond with each --volume ?

Reply:
@cdoern No, a standard volume option - so `-v volname:/path/in/container:subpath=/sub/path/in/volume,ro,Z`